### DISCUSSION TITLE
启动的docker命令里漏掉了6800

### DISCUSSION THREAD

**Discussion by imbiss** [AlistGo/alist#3745] [General]
按照页面提示的方法启动发现无法链接aria的6800端口， 改为以下就好了

`docker run -d --restart=always -v /etc/alist:/opt/alist/data -p 5244:5244 **-p 6800:6800** -e PUID=0 -e PGID=0 -e UMASK=022 --name="alist" xhofe/alist-aria2:latest`

**Comment by imbiss** [comment]
上面语句内注意去掉**

**Reply by mackon666** [reply]
我用的也是这个镜像，请问如何进入aria2的管理页面，能设置下载到外接硬盘吗

**Reply by imbiss** [reply]
aria2的设置不在alist里。你需要安装一个aria2的前端。比如ariaNg.通过6800端口管理aria2.在ariaNg里的aria2
设置里，可以把下载路径定义到暴露在容器外的路径。  
我估计，如果是活动硬盘，要先要挂载活动硬盘，然后启动镜像的时候确保该路径在容器内能访问。应该就没问题。

**Reply by Turkeyer** [reply]
xhofe/alist-aria2已经打包了aria2，应该不用再安装NG管理，之前的版本是可以直接使用的。

**Reply by imbiss** [reply]
没懂，请问何直接使用aria2？ 我是打开了6800端口以后，才能从容器外管理aria2的，然后才能下载到指定目录的。
